### DISCUSSION TITLE
同時押しが反応しないキーが2つある

### DISCUSSION THREAD

**Discussion by k-fujishiro-dev** [Yowkees/keyball#626] [Q&A]
### 不具合状況
・左手側キーボードの赤色で囲った箇所のキーが同時押しに反応しません
![keyball_左表](https://github.com/user-attachments/assets/27731f97-ae3d-44ac-9912-67825122f8fd)

・例として、Ctrl＋aや、Alt＋Tabなどが反応しません

### 試したこと
ファームウェアの再書き込み

### はんだ付け状態の写真
赤丸のキーが不具合が発生している箇所になります。
![keyball_左裏](https://github.com/user-attachments/assets/7dabf0e4-839f-43cd-9238-4320b9f97695)

### 環境
device: keyball39
os: windows 11
firmware: Keyball39_via_v1.3.2

**Comment by k-fujishiro-dev** [comment]
remapで同時押しに対応していないtapdキーを選択していました。
通常のCtrl、Altキーを選択したところ同時押しが可能になりました。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。ご自身で解決していただき、誠にありがとうございます。

ぜひ末永くご愛用いただければ幸いです。
